doc/release-notes.md

```diff
@@ -1,6 +1,9 @@
-Bitcoin Core version 27.x is now available from:
+27.1rc1 Release Notes
+=====================
 
-  <https://bitcoincore.org/bin/bitcoin-core-27.x/>
+Bitcoin Core version 27.1 is now available from:
+
+  <https://bitcoincore.org/bin/bitcoin-core-27.1/>
```

Comment:
```suggestion
Bitcoin Core version 27.1rc1 is now available from:

  <https://bitcoincore.org/bin/bitcoin-core-27.1/test.rc1/>
```

Reply:
nit but text is not updated to "271.rc1" in latest force push (link is fine now)